### PR TITLE
bar_14

### PR DESCRIPTION
**Pull request by erinlincoln**
- Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/bar_14.ncl
- Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/bar_14_lg.png 
- Issue #341 
- 
image

### CODE REVIEW

File: Plots/Bar/NCL_bar_14.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# Plot:
+
+# Create figure with figure size (width, height) in inches and create axes
+fig = plt.figure(figsize=(6, 5.25))
+ax = plt.axes()
+
+# Plot the bar chart, line chart, and a horizontal line with varying colors and linewidths
+plt.bar(months, y1, color="yellow", edgecolor="k", linewidth=.5)
+plt.plot(months, y2, color="b", linewidth=1)
+plt.plot([0, 13], [40, 40], color="b", linewidth=1)
```
Comment (anissa111) [inline Plots/Bar/NCL_bar_14.py:45]: There's actually a way to plot horizontal lines in matplotlib. See the [hlines documentation](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.hlines.html).

**Review by anissa111 — COMMENTED** [review]
This looks really good! I left a note of something you might want to consider.

File: Plots/Bar/NCL_bar_14.py (hunk 1 above)
Comment (erinlincoln) [reply Plots/Bar/NCL_bar_14.py]: Just fixed it to use hlines down below! Thanks for the help!

File: Plots/Bar/NCL_bar_14.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
+"""
+NCL_bar_14.py
+===========
```
Comment (michaelavs) [inline Plots/Bar/NCL_bar_14.py:3]:
```suggestion
=============
```

File: Plots/Bar/NCL_bar_14.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+# Use geocat.viz.util convenience function to add major and minor tick lines
+gvutil.add_major_minor_ticks(ax, x_minor_per_major=3, y_minor_per_major=4)
+
+# Adjust tick and ticklabel sizes for each axis
+ax.tick_params(axis="x", size=12, labelsize=12)
+ax.tick_params(axis="y", size=12, labelsize=12)
+axRHS.tick_params(axis="y", size=12, labelsize=12)
+
+#Display plot
```
Comment (michaelavs) [inline Plots/Bar/NCL_bar_14.py:81]:
```suggestion
# Display plot
```

**Comment by michaelavs** [comment]
This looks good to me! I noted two minor formatting items and once they are addressed, I will approve this PR!

**Review by michaelavs — APPROVED** [review]

**Pull request merged**